pages/common/iverilog.md

```diff
@@ -1,24 +1,24 @@
 # iverilog
 
-> Preprocesses and compiles Verilog HDL (IEEE-1364) code, into executable programs for simulation.
-> More information: <http://iverilog.icarus.com/>.
+> Preprocesses and compiles Verilog HDL (IEEE-1364) code into executable programs for simulation.
+> More information: <https://github.com/steveicarus/iverilog>.
```

Comment:
Updating this link because their repo has examples and documentation, I could have directly linked to https://steveicarus.github.io/iverilog/ page but I think linking to Repo is better here. If necessary I can change it.